### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by ThomasLandauer**

### CODE REVIEW

**Comment by ThomasLandauer** [comment]
I also changed the "sample" setting to `defaultFont`, since `defaultPaperSize` isn't working (see https://github.com/nucleos/NucleosDompdfBundle/issues/1)

File: README.md [hunk 1]
```diff
@@ -53,7 +53,9 @@ final class MyService
     public function render()
     {
         // ...
-        $this->factory->create();
+        $dompdf = $this->factory->create(); // Returns a `Dompdf\Dompdf` object
```
Comment (core23) [inline README.md:56]: The `Returns ...` comment seems redundant to me

File: README.md [hunk 2]
```diff
@@ -71,6 +73,7 @@ final class MyOtherService
         $html = '<h1>Sample Title</h1><p>Lorem Ipsum</p>';
 
         $response = $this->wrapper->getStreamResponse($html, "document.pdf");
+        // `$response` is a `Symfony\Component\HttpFoundation\StreamedResponse`
```
Comment (core23) [inline README.md:76]: Same here

**Review by core23 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ThomasLandauer) [reply README.md:56]: Well, to me it wasn't clear what a method called `create()` could return ;-)

File: README.md (hunk 2 above)
Comment (ThomasLandauer) [reply README.md:76]: In this case it wasn't clear that it's a **Symfony** StreamedResponse.

Comment (core23) [reply README.md:76]: Then you should use `// @var Symfony\Component\HttpFoundation\StreamedResponse` as this is standard phpdoc for type information.

File: README.md (hunk 1 above)
Comment (core23) [reply README.md:56]: This is just example code to get started. I would leave out anything special that a normal developer could not figure out itself.

Comment (ThomasLandauer) [reply README.md]: If it's explained in README anybody can figure it out *before* installing it. Besides, one additional line here doesn't hurt anybody, and the `@var` annotation is a really nice solution :-)

**Comment by core23** [comment]
Thanks @ThomasLandauer

**Pull request merged**